Lib/test/test_dis.py

```diff
@@ -390,6 +398,42 @@ def foo(x):
        _h.__code__.co_firstlineno + 3,
 )
 
+
+
+@contextlib.contextmanager
+def ignore_coroutine_never_awaited_warnings():
```

Comment:
@serhiy-storchaka, @1st1 Review question here - for a dis module test, we want to create a coroutine that we never await, so @syncosmic has suggested adding this helper context manager.

I like the idea, but I'm wondering if we should put it in test.support, rather than inline in the dis module tests.

Thoughts?